app/analysis/edit/controller.js

```diff
@@ -87,58 +101,46 @@ export default Controller.extend(AnalysisEditQueryParams.Mixin, {
       model: this.store.peekRecord("report-intersection", res.data.id),
       meta: res.meta,
     };
-  }),
+  }
 
-  _customer: computed("intersectionModel.customer.id", "store", function () {
-    const id = this.get("intersectionModel.customer.id");
+  get _customer() {
+    const id = get(this, "intersectionModel.customer.id");
     return id && this.store.peekRecord("customer", id);
-  }),
+  }
 
-  _project: computed("intersectionModel.project.id", "store", function () {
-    const id = this.get("intersectionModel.project.id");
+  get _project() {
+    const id = get(this, "intersectionModel.project.id");
```

Comment:
`get` necessary or just easier refactoring?

Reply:
Necessary, since we are accessing a proxy.